bin/remove-test-dbs.js

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+
+const settingsFilePath = './settings.json';
+const settingsFile = fs.existsSync(settingsFilePath) ? settingsFilePath : './settings.json.default.json';
+const settings = JSON.parse(fs.readFileSync(settingsFile)).nightwatch;
+const dbUrl = `http://${settings.fauxton_username}:${settings.password}@${settings.db_host}:${settings.db_port}`;
+const nano = require('nano')(dbUrl);
+
+nano.db.list((err, body) => {
+  if (err) {
+    console.log('ERR', err);
+    return;
+  }
+  const out = body.forEach(db => {
```

Comment:
the forEach loop handles async code (removing the dbs) in a sync fashion and logs error related output to stdout instead of stderr. to fix that, use console.error. for the sync issue you can use https://github.com/caolan/async/blob/v1.5.2/README.md#parallel (we require the module async already in other parts of our build-related infrastructure).

example (not tested):

```js
const list = body.filter(db => {
  return /fauxton-selenium-tests/.test(db);
});

list.map(db => {
  return function (err, cb) {
    nano.db.destroy(db, (err, resp) => {
      if (err) {
        return cb(err);
      }
      cb(null);
    })
  }
});

async.parallel(list, () => {
  if (err) {
    console.error(err);
  }
})
```